Libs/MRML/Widgets/Resources/UI/qMRMLWindowLevelWidget.ui

```diff
@@ -19,46 +19,27 @@
   <property name="windowTitle">
    <string>Window Level</string>
   </property>
-  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0,0">
+  <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
-    <widget class="ctkDoubleSpinBox" name="WindowSpinBox">
-     <property name="prefix">
-      <string>W: </string>
-     </property>
-     <property name="decimals">
-      <number>2</number>
-     </property>
-     <property name="decimalsOption">
-      <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Window/Level:</string>
```

Comment:
@lassoan Regarding this new string and the new tooltips added in this PR, am I missing anything related to marking it for language translating? I just realized that I didn’t do anything for it. Do I need to do anything for strings in .ui files?

Reply:
All strings properties in .ui files are translatable by default, unless marked explicitly as non-translatable. So, there is probably nothing to do. (Other than updating the translation source files on Weblate - that I'll do in a few days).

Reply:
Ok thanks for the info!